locales/hin.json

```diff
@@ -268,7 +268,8 @@
   "MyVcsTab": {
     "downloadCard": "डाउनलोड कार्ड",
     "bringYourDigitalID": "अपनी डिजिटल आईडी लाओ",
-    "generateVcDescription": "अपना कार्ड डाउनलोड करने के लिए नीचे डाउनलोड कार्ड टैप करें",
+    "generateVcDescription": "अपना कार्ड डाउनलोड करने के लिए नीचे + कार्ड टैप करें",
+    "generateVcFABDescription": "अपना कार्ड डाउनलोड करने के लिए नीचे डाउनलोड कार्ड टैप करें",
```

Comment:
value needs to be swapped
"generateVcDescription": "अपना कार्ड डाउनलोड करने के लिए नीचे डाउनलोड कार्ड टैप करें",
"generateVcFABDescription": "अपना कार्ड डाउनलोड करने के लिए नीचे + कार्ड टैप करें",

Reply:
as a part of clean up task of removing the feature toggle, we will be removing it.